docs/src/modules/components/ApiPage/list/ClassesList.tsx

```diff
@@ -42,8 +42,9 @@ type ClassesListProps = {
 };
 type HashParams = { componentName?: string; className: string };
 
-export const getHash = ({ componentName, className }: HashParams) =>
-  `${componentName ? `${componentName}-` : ''}classes-${className}`;
+export function getHash({ componentName, className }: HashParams) {
+  return `${componentName ? `${componentName}-` : ''}classes-${className}`;
```

Comment:
I'm not sure about this logic. This feels weird: https://mui.com/material-ui/api/dialog/#Dialog-classes-paper
I think https://mui.com/material-ui/api/dialog/#.MuiDialog-paper would be for the best DX.

Reply:
I wrote it with the goal of having unique `id` in mind. So I went with the strategy: 

```
id = [componentName]-[listType]-[lineName]
```

The extreme case is the base API pages where multiple components are documented in a single web page. Here for example are two instance of `.Mui-expanded` in the same page:

https://mui.com/base-ui/react-menu/components-api/#MenuButton-classes-expanded
https://mui.com/base-ui/react-menu/components-api/#Menu-classes-expanded

Make me realise the navigation is broken for css classes

Reply:
Ah right, I forgot about these global class names 🤦‍♂️, they are not unique